### PR TITLE
3.85+

### PR DESCRIPTION
**Pull request by Zhy54**
This is my first time upload a file to github, I don't know I'm correct or not.

### CODE REVIEW

File: Rapidrive.zh-Hans.xlf [hunk 1]
```diff
@@ -1,5003 +1,5003 @@
...
+          <source>Pressing the above hotkey from any program should:</source>
+          <target state="translated">从以上程序中按下上述快捷键应：</target>
+        </trans-unit>
+        <trans-unit id="FileOperationsHandler" translate="yes" xml:space="preserve">
+          <source>File operations handler</source>
+          <target state="translated">文件操作处理程序</target>
+        </trans-unit>
+        <trans-unit id="FileOperationsHandlerInfo" translate="yes" xml:space="preserve">
+          <source>Choice between default (Explorer), TeraCopy (must be already installed), and Taskmaster (New)</source>
+          <target state="translated">在默认（资源管理器）、TeraCopy（必须已安装）、以及Taskmaster（新增）之间进行选择</target>
```
Comment (sunxyw) [inline Rapidrive.zh-Hans.xlf:887]: 如果英文和中文之间可以加入空格会更好。

**Review by sunxyw — APPROVED** [review]
你好，我是繁体中文翻译的主要贡献者，非常感谢你的贡献。大部分内容在我看来很好，只是我注意到你好像更改了空格或换行符的格式，导致有大量无关的diff，如果可以消除就更好了。


Hi, I'm the main contributor of the Traditional Chinese localization, thank you very much for your contribution. Most of it looks fine to me, except I noticed that you seem to have changed the formatting of spaces or line breaks, resulting in a lot of extraneous diffs, which would be nice if you can remove them.

File: Rapidrive.zh-Hans.xlf (hunk 1 above)
Comment (Zhy54) [reply Rapidrive.zh-Hans.xlf:887]: > 如果英文和中文之间可以加入空格会更好。

您好！我不太会用GitHub，我发现大部分是中文和英文之间是有空格的，我改的几个都是从无空格改到有空格。
另外补充了未翻译的文本，其实未翻译的部分很少

Comment (Zhy54) [reply Rapidrive.zh-Hans.xlf:887]: > 如果英文和中文之间可以加入空格会更好。

唯一一个没加空格的就是这个887，上一个版本此处为“任务大师”，考虑到软件无公认中文名，于是还原为英文，前后有大量的全角符号、（）等，我怕显得太空就没加空格

**Comment by m1l** [comment]
For some reason it shows as if every line was changed, but I think I can import it regardless. I'll publish it with 3.90.1. Thanks!

**Pull request merged**